cg/cli/demultiplex/demux.py

```diff
@@ -220,3 +228,16 @@ def confirm_flow_cell_sync(context: CGConfig, source_directory: str):
                 source_flow_cell.name,
                 DemultiplexingDirsAndFiles.COPY_COMPLETE,
             ).touch()
+
+
+@click.command(name="create-manifest-files")
+@click.option(
+    "-s",
+    "--source-directory",
+    required=True,
+    help="The path from where the syncing is done.",
+)
+def create_manifest_files(source_directory: str):
+    for source_flow_cell in Path(source_directory).iterdir():
+        if needs_manifest_file(source_flow_cell):

```

Comment:
```suggestion
        if is_manifest_file_required(source_flow_cell):

```
Since a bool is returned